project/profiles/capacity4more/themes/c4m/kapablo/theme/preprocess.inc

```diff
@@ -616,8 +616,8 @@ function kapablo_preprocess_html(&$variables) {
  * Implements template_preprocess_user_profile().
  */
 function kapablo_preprocess_user_profile(&$variables) {
-  $blocks = array(block_load('c4m_user_profile', 'user_my_profile_groups'));
-  $renderable_blocks = _block_render_blocks($blocks);
+  $block = block_load('c4m_user_profile', 'user_my_profile_groups');
```

Comment:
change it in here:
https://github.com/capacity4dev/capacity4more/pull/1155/files#diff-dcea7c519f8e4ed9be3a183c61bc4ff6R619